app/_src/gateway/kong-enterprise/partials.md

```diff
@@ -0,0 +1,214 @@
+
+## Using a Partial in Plugin Configuration
+
+After creating a Partial, you can link it to a plugin using a POST or PUT request. For example, to link a rate-limiting plugin to a Partial:
+
+```sh
+curl -i -X POST http://{HOSTNAME}:8001/plugins \
+  -H "Content-Type: application/json" \
+  --data '{
+    "name": "rate-limiting",
+    "config": {
+      "minute": 10
+    },
+    "partials": [
+      { "id": "PARTIAL-ID-TO-LINK" }
+    ]
+  }'
+```
+
+### Important Notes:
+- You cannot provide inline Redis configuration alongside Partial-linked configurations. Either define Redis settings in the plugin or skip the Redis key and use a Partial.
+
+- The response will include the plugin's configuration along with the expanded Partial configuration. Together, these represent the complete configuration the plugin will use, merging its individual settings with those inherited from the linked Partial. This ensures you can see exactly how the plugin will behave.
+
+
+Example response:
+```sh
+HTTP/1.1 201 Created
+Access-Control-Allow-Origin: *
+Connection: keep-alive
+Content-Length: 1021
+Content-Type: application/json; charset=utf-8
+Date: Thu, 06 Mar 2025 17:21:56 GMT
+Server: kong/3.10.0.0-enterprise-edition
+X-Kong-Admin-Latency: 18
+X-Kong-Admin-Request-ID: ecae700275f3aa1a7f48f4483305c465
+
+{
+    "config": {
+        "day": null,
+        "error_code": 429,
+        "error_message": "API rate limit exceeded",
+        "fault_tolerant": true,
+        "header_name": null,
+        "hide_client_headers": false,
+        "hour": null,
+        "limit_by": "consumer",
+        "minute": 10,
+        "month": null,
+        "path": null,
+        "policy": "redis",
+        "redis": {
+            "database": 0,
+            "host": "localhost",
+            "password": null,
+            "port": 6379,
+            "server_name": null,
+            "ssl": false,
+            "ssl_verify": false,
+            "timeout": 2000,
+            "username": null
+        },
+        "redis_database": 0,
+        "redis_host": "localhost",
+        "redis_password": null,
+        "redis_port": 6379,
+        "redis_server_name": null,
+        "redis_ssl": false,
+        "redis_ssl_verify": false,
+        "redis_timeout": 2000,
+        "redis_username": null,
+        "second": null,
+        "sync_rate": -1,
+        "year": null
+    },
+    "consumer": null,
+    "consumer_group": null,
+    "created_at": 1741281716,
+    "enabled": true,
+    "id": "974a63f3-87e9-4f6b-ace8-5e54c9084760",
+    "instance_name": null,
+    "name": "rate-limiting",
+    "ordering": null,
+    "partials": [
+        {
+            "id": "a542d158-b73f-4bb7-921e-ab1debc772a0",
+            "name": "my-ce-partial-1",
+            "path": "config.redis"
+        }
+    ],
+    "protocols": [
+        "grpc",
+        "grpcs",
+        "http",
+        "https"
+    ],
+    "route": null,
+    "service": null,
+    "tags": null,
+    "updated_at": 1741281716
+}
+```
```

Comment:
```suggestion
```
We don't need to do this in the docs. If anything, it should be in the API spec.